python-avocado.spec

```diff
@@ -757,14 +757,14 @@ Some of them are used as functional tests of the framework, others serve as
 examples of how to write tests on your own.
 
 %files examples
-%dir %{_datadir}/avocado
-%{_datadir}/avocado/gdb-prerun-scripts
-%{_datadir}/avocado/plugins
-%{_datadir}/avocado/tests
-%{_datadir}/avocado/wrappers
-%{_datadir}/avocado/yaml_to_mux
-%{_datadir}/avocado/yaml_to_mux_loader
-%{_datadir}/avocado/varianter_pict
+%dir %{_docdir}/avocado
+%{_docdir}/avocado/gdb-prerun-scripts
+%{_docdir}/avocado/plugins
+%{_docdir}/avocado/tests
+%{_docdir}/avocado/wrappers
+%{_docdir}/avocado/yaml_to_mux
+%{_docdir}/avocado/yaml_to_mux_loader
+%{_docdir}/avocado/varianter_pict
```

Comment:
I'd expect this to include `setup.py` or `MANIFEST` changes, but I looked at the `pypi` and `make install` installation and the the `examples` are not included, despite the `MANIFEST`. @clebergnu do we want to do anything about that?